### ISSUE TITLE
`get_instruction_text` in disassembler.py can return None, leading to errors

### ISSUE THREAD

**Issue by d0mnik** [Njord0/BNPyc#2]
`get_instruction_text` in `disassembler.py` can return None in its list of `InstructionTextToken`, leading to `AttributeError`.

```
Traceback (most recent call last):
  File "/Applications/dev-binja.app/Contents/MacOS/plugins/../../Resources/python/binaryninja/architecture.py", line 676, in _get_instruction_text
    token_buf = function.InstructionTextToken._get_core_struct(tokens)
  File "/Applications/dev-binja.app/Contents/MacOS/plugins/../../Resources/python/binaryninja/architecture.py", line 2842, in _get_core_struct
    result[j].type = tokens[j].type
AttributeError: 'NoneType' object has no attribute 'type'
```

This leads to failed disassembly as shown:

![image](https://user-images.githubusercontent.com/32241418/228829192-4ad0a3a6-3907-4c0c-a22d-fbf63ff0bed3.png)

**Comment by d0mnik** [comment]
File in question:
[ctf-challenge.zip](https://github.com/Njord0/BNPyc/files/11111124/ctf-challenge.zip)

**Comment by Njord0** [comment]
The bug source was in the `add_jump` method returning `None` for unhandled jump instructions, fixed with commits (dad35c6 and 520020f).

I also fixed absolute and relative jumps, the graph is more accurate now.

**Comment by d0mnik** [comment]
Just tested with latest commit, all instructions appear to be handled!
